pkg/flow/engine.go

```diff
@@ -164,9 +164,9 @@ func (engine *engine) NewInstance(ctx context.Context, args *newInstanceArgs) (*
 
 	t := time.Now()
 	engine.pubsub.NotifyInstances(d.ns())
-	engine.logToNamespace(ctx, t, d.ns(), "Workflow '%s' has been triggered by %s.", args.Path, args.Caller)
-	engine.logToWorkflow(ctx, t, d.wfData, "Instance '%s' created by %s.", im.ID().String(), args.Caller)
-	engine.logToInstance(ctx, t, in, "Preparing workflow triggered by %s.", args.Caller)
+	engine.tagLogToNamespace(ctx, t, d, "Workflow '%s' has been triggered by %s.", args.Path, args.Caller)
+	engine.tagLogToWorkflow(ctx, t, d, "Instance '%s' created by %s.", im.ID().String(), args.Caller)
+	engine.tagLogToInstance(ctx, t, im, "Preparing workflow triggered by %s.", args.Caller)
```

Comment:
I understand changing the `tagLogToInstance` function to take slightly different arguments. But why was it necessary to rename the function entirely? Why not just leave it as `logToInstance`? The caller probably has no interest in having a choice between the two. If all logs should be tagged, then the caller shouldn't have to be aware of the change. Long-term, I think this rename makes the function a little bit more confusing and causes possible unnecessary merge conflicts.

Reply:
Looks like this has been resolved.